--- ext-xdebug.ini.default	2017-05-16 11:30:53.000000000 +0800
+++ ext-xdebug.ini	2017-05-16 19:52:20.000000000 +0800
@@ -1,2 +1,14 @@
 [xdebug]
 zend_extension="/usr/local/opt/php56-xdebug/xdebug.so"
+xdebug.remote_enable=1
+xdebug.remote_mode=req 
+xdebug.remote_handler=dbgp
+xdebug.remote_host=drupal.xyz
+xdebug.remote_port=9000
+xdebug.auto_trace=on
+xdebug.collect_params=on
+xdebug.collect_return=on
+xdebug.max_nesting_level=256
+xdebug.profile_enable=on
+xdebug.trace_output_dir="/tmp"
+xdebug.profile_output_dir="/tmp"
