--- php.ini.default	2016-10-19 09:30:40.000000000 +0800
+++ php.ini	2016-12-06 21:44:26.000000000 +0800
@@ -369,7 +369,7 @@
 ; Maximum execution time of each script, in seconds
 ; http://php.net/max-execution-time
 ; Note: This directive is hardcoded to 0 for the CLI SAPI
-max_execution_time = 30
+max_execution_time = 300
 
 ; Maximum amount of time each script may spend parsing request data. It's a good
 ; idea to limit this time on productions servers in order to eliminate unexpectedly
@@ -657,7 +657,7 @@
 ; Its value may be 0 to disable the limit. It is ignored if POST data reading
 ; is disabled through enable_post_data_reading.
 ; http://php.net/post-max-size
-post_max_size = 8M
+post_max_size = 64M
 
 ; Automatically add files before PHP document.
 ; http://php.net/auto-prepend-file
@@ -817,10 +817,10 @@
 
 ; Maximum allowed size for uploaded files.
 ; http://php.net/upload-max-filesize
-upload_max_filesize = 2M
+upload_max_filesize = 32M
 
 ; Maximum number of files that can be uploaded via a single request
-max_file_uploads = 20
+max_file_uploads = 9
 
 ;;;;;;;;;;;;;;;;;;
 ; Fopen wrappers ;
@@ -845,7 +845,7 @@
 
 ; Default timeout for socket based streams (seconds)
 ; http://php.net/default-socket-timeout
-default_socket_timeout = 60
+default_socket_timeout = 600
 
 ; If your scripts have to deal with files from Macintosh systems,
 ; or you are running on a Mac and need to deal with files from
@@ -934,12 +934,15 @@
 ; Defines the default timezone used by the date functions
 ; http://php.net/date.timezone
 ;date.timezone =
+date.timezone = Asia/Shanghai
 
 ; http://php.net/date.default-latitude
 ;date.default_latitude = 31.7667
+date.default_latitude = 34.5000
 
 ; http://php.net/date.default-longitude
 ;date.default_longitude = 35.2333
+date.default_longitude = 121.4333
 
 ; http://php.net/date.sunrise-zenith
 ;date.sunrise_zenith = 90.583333
