--- httpd.conf.default	2018-04-16 02:09:12.000000000 +0800
+++ httpd.conf	2018-04-16 08:55:55.000000000 +0800
@@ -172,8 +172,8 @@
 #LoadModule speling_module libexec/apache2/mod_speling.so
 #LoadModule userdir_module libexec/apache2/mod_userdir.so
 LoadModule alias_module libexec/apache2/mod_alias.so
-#LoadModule rewrite_module libexec/apache2/mod_rewrite.so
-#LoadModule php7_module libexec/apache2/libphp7.so
+LoadModule rewrite_module libexec/apache2/mod_rewrite.so
+LoadModule php7_module libexec/apache2/libphp7.so
 #LoadModule perl_module libexec/apache2/mod_perl.so
 LoadModule hfs_apple_module libexec/apache2/mod_hfs_apple.so
 
@@ -218,6 +218,7 @@
 # If your host doesn't have a registered DNS name, enter its IP address here.
 #
 #ServerName www.example.com:80
+ServerName localhost
 
 #
 # Deny access to the entirety of your server's filesystem. You must
