--- /etc/php-fpm.conf	2018-04-25 14:42:41.000000000 +0800
+++ php-fpm.conf	2018-04-25 14:45:54.000000000 +0800
@@ -68,7 +68,7 @@
 ; Available units: s(econds), m(inutes), h(ours), or d(ays)
 ; Default Unit: seconds
 ; Default Value: 0
-process_control_timeout = 1
+;process_control_timeout = 0
 
 ; The maximum number of processes FPM will fork. This has been design to control
 ; the global number of processes when using dynamic PM within a lot of pools.
@@ -445,7 +445,7 @@
 ; dumped to the 'slowlog' file. A value of '0s' means 'off'.
 ; Available units: s(econds)(default), m(inutes), h(ours), or d(ays)
 ; Default Value: 0
-;request_slowlog_timeout = 0
+request_slowlog_timeout = 1
  
 ; The timeout for serving a single request after which the worker process will
 ; be killed. This option should be used when the 'max_execution_time' ini option
