--- nginx.conf.orig	2016-10-19 00:55:11.000000000 +0800
+++ nginx.conf	2016-12-06 21:18:50.508168474 +0800
@@ -1,8 +1,18 @@
 
-user  nginx;
-worker_processes  1;
+user  www-data;
+worker_processes  2;
 
+# @see http://nginx.org/en/docs/ngx_core_module.html#error_log
+#      log level: debug | info | notice | warn | error ...
+# for debuging
+# error_log  /var/log/nginx/error.log debug;
+
+# for small site
 error_log  /var/log/nginx/error.log warn;
+
+# for production site
+# error_log  /var/log/nginx/error.log error | crit;
+
 pid        /var/run/nginx.pid;
 
 
@@ -19,14 +29,21 @@
                       '$status $body_bytes_sent "$http_referer" '
                       '"$http_user_agent" "$http_x_forwarded_for"';
 
+    # http://nginx.org/en/docs/http/ngx_http_log_module.html#access_log
+    # for small site or debugging
     access_log  /var/log/nginx/access.log  main;
 
+    # for heavy load production site
+    # access_log off
+
     sendfile        on;
     #tcp_nopush     on;
 
+    #keepalive_timeout  0;
     keepalive_timeout  65;
 
-    #gzip  on;
+    gzip  on;
+    gzip_disable  "MSIE [1-6]\.(?!.*SV1)";
 
     include /etc/nginx/conf.d/*.conf;
 }
