--- 000-default.conf.orig	2018-03-30 23:07:14.000000000 +0800
+++ 000-default.conf	2018-04-17 21:10:35.222004652 +0800
@@ -11,14 +11,20 @@
 	ServerAdmin webmaster@localhost
 	DocumentRoot /var/www/html
 
+ 	<Directory /var/www/html>
+ 		Options Indexes FollowSymLinks MultiViews
+		AllowOverride all
+		Require all granted
+	</Directory>
+
 	# Available loglevels: trace8, ..., trace1, debug, info, notice, warn,
 	# error, crit, alert, emerg.
 	# It is also possible to configure the loglevel for particular
 	# modules, e.g.
 	#LogLevel info ssl:warn
 
-	ErrorLog ${APACHE_LOG_DIR}/error.log
-	CustomLog ${APACHE_LOG_DIR}/access.log combined
+	ErrorLog "| /usr/bin/cronolog ${APACHE_LOG_DIR}/error-%Y%m%d.log"
+	CustomLog "| /usr/bin/cronolog ${APACHE_LOG_DIR}/access-%Y%m%d.log" combined
 
 	# For most configuration files from conf-available/, which are
 	# enabled or disabled at a global level, it is possible to
